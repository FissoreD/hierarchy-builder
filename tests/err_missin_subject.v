From HB Require Import structures.
HB.mixin Record M X := {}.
HB.structure Definition S := { X of M X}.
HB.instance Definition _ : M nat := M.Build _.
HB.instance Definition _ : M _ := M.Build bool.
Fail HB.instance Definition _ : M _ := M.Build _.