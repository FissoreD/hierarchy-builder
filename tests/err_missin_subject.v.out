HB: The command has indeed failed with message:
The instance subject must be explicitly given.
Use:
  HB.instance Definition _ : M <subject> := ...
  HB.instance Definition _ := M.Build <subject> ...
